exporter/jaeger/jaegergrpcexporter/exporter.go

```diff
@@ -12,58 +12,67 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package jaegerexporter
+package jaegergrpcexporter
 
 import (
 	"context"
 
-	"go.uber.org/zap"
-	"google.golang.org/grpc"
-
 	jaegerproto "github.com/jaegertracing/jaeger/proto-gen/api_v2"
+	"google.golang.org/grpc"
 
-	"github.com/open-telemetry/opentelemetry-service/consumer"
 	"github.com/open-telemetry/opentelemetry-service/consumer/consumerdata"
 	"github.com/open-telemetry/opentelemetry-service/consumer/consumererror"
+	"github.com/open-telemetry/opentelemetry-service/exporter"
+	"github.com/open-telemetry/opentelemetry-service/exporter/exporterhelper"
 	jaegertranslator "github.com/open-telemetry/opentelemetry-service/translator/trace/jaeger"
 )
 
-// JaegerProtoGRPCSender forwards spans encoded in the jaeger proto
-// format, to a grpc server.
-type JaegerProtoGRPCSender struct {
-	client jaegerproto.CollectorServiceClient
-	logger *zap.Logger
-}
-
-var _ consumer.TraceConsumer = (*JaegerThriftHTTPSender)(nil)
-
-// NewJaegerProtoGRPCSender returns a new GRPC-backend span sender.
-// The collector endpoint should be of the form "hostname:14250".
-func NewJaegerProtoGRPCSender(collectorEndpoint string, zlogger *zap.Logger) *JaegerProtoGRPCSender {
+// New returns a new Jaeger gRPC exporter.
+// The exporter name is the name to be used in the observability of the exporter.
+// The collectorEndpoint should be of the form "hostname:14250" (a gRPC target).
+func New(exporterName, collectorEndpoint string) (exporter.TraceExporter, error) {
```

Comment:
Nit: do we want to use `zap.Logger` in this exporter?

Reply:
At this stage the observability is covered via metrics but the main point is to do any added observability on the exporterhelper. Notice that on the typical usage that includes a queued-retry processor there will be logging for errors and retries.